objects/omapcamera/Create_0.gml

```diff
@@ -0,0 +1,10 @@
+camWidth = 160;
```

Comment:
Use the full word, never abbreviate.

```camWidth``` should be ```cameraWidth```

Same for Height just below this.